projects/plugins/jetpack/modules/subscriptions/views.php

```diff
@@ -210,43 +210,36 @@ static function render_widget_status_messages( $instance ) {
-					echo "</div>";
+					$message = __( 'You already have several pending email subscriptions. Approve or delete a few through your <a href="https://subscribe.wordpress.com/">Subscription Manager</a> before attempting to subscribe to more blogs.', 'jetpack' );
 					break;
 				case 'spammed':
-					echo "<div style='background-color: #{$themecolors['bg']}; border: 1px solid #{$themecolors['border']}; color: #{$themecolors['text']}; padding-left: 5px; padding-right: 5px; margin-bottom: 10px;'>";
-					echo wp_kses_post( sprintf( __( 'Because there are many pending subscriptions for this email address, we have blocked the subscription. Please <a href="%s">activate or delete</a> pending subscriptions before attempting to subscribe.' ), 'https://subscribe.wordpress.com/' ) );
-					echo "</div>";
+					/* translators: %s is a URL */
+					$message = sprintf( __( 'Because there are many pending subscriptions for this email address, we have blocked the subscription. Please <a href="%s">activate or delete</a> pending subscriptions before attempting to subscribe.', 'jetpack' ), 'https://subscribe.wordpress.com/' );
 					break;
 				case 'subscribed':
-					echo "<div style='background-color: #{$themecolors['bg']}; border: 1px solid #{$themecolors['border']}; color: #{$themecolors['text']}; padding-left: 5px; padding-right: 5px; margin-bottom: 10px;'>";
-					_e( 'You&rsquo;re already subscribed to this site.' );
-					echo "</div>";
+					$message = __( 'You&rsquo;re already subscribed to this site.', 'jetpack' );
 					break;
 				case 'pending':
-					echo "<div style='background-color: #{$themecolors['bg']}; border: 1px solid #{$themecolors['border']}; color: #{$themecolors['text']}; padding-left: 5px; padding-right: 5px; margin-bottom: 10px;'>";
-					_e( 'You have a pending subscription already; we just sent you another email. Click the link or <a href="https://en.support.wordpress.com/contact/">contact us</a> if you don&rsquo;t receive it.' );
-					echo "</div>";
+					$message = __( 'You have a pending subscription already; we just sent you another email. Click the link or <a href="https://en.support.wordpress.com/contact/">contact us</a> if you don&rsquo;t receive it.', 'jetpack' );
 					break;
 				case 'confirmed':
-					echo "<div style='background-color: #{$themecolors['bg']}; border: 1px solid #{$themecolors['border']}; color: #{$themecolors['text']}; padding-left: 5px; padding-right: 5px; margin-bottom: 10px;'>";
-					_e( 'Congrats, you&rsquo;re subscribed! You&rsquo;ll get an email with the details of your subscription and an unsubscribe link.' );
-					echo "</div>";
+					$message = __( 'Congrats, you&rsquo;re subscribed! You&rsquo;ll get an email with the details of your subscription and an unsubscribe link.', 'jetpack' );
 					break;
 			}
+
+			echo sprintf( "<div style='border: 1px solid #%s; padding-left: 5px; padding-right: 5px; margin-bottom: 10px;'>", esc_attr( $themecolors['border'] ) );
```

Comment:
During my tests (using Spearhead) I noticed that, where `$themecolors['border']` is empty, the following invalid CSS is output:

`border: 1px solid #; padding-left: 5px; padding-right: 5px; margin-bottom: 10px;`

Do you think it'd be better to only print the inline style `border: 1px solid #; ` if `$themecolors['border']` is empty?

Reply:
Agreed 👍 I kept the `1px solid` border but removed the empty color value, rather than omitting the border altogether. The border takes on the text-color in this case.

<img width="835" alt="Screen Shot 2021-03-22 at 2 05 52 PM" src="https://user-images.githubusercontent.com/63313398/112058498-e8bad880-8b17-11eb-833d-43c54d1474a3.png">

Per the discussion below -- once we can access the theme border color with CSS variables, we can move all of the styling out into the stylesheets and it will be much cleaner :)